term : remove "Dvcfile" from versioning use case tutorial (#1526)
iterative/dvc.org: content/docs/use-cases/versioning-data-and-model-files/tutorial.md

> ```

```dvc
$ dvc run -n train -d train.py -d data \
          -o model.h5 -o bottleneck_features_train.npy \
          -o bottleneck_features_validation.npy -M metrics.csv \
          python train.py
```

> `dvc run` creates a pipeline stage named `train` (specified using the `-n`
> option) in [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file)
> file. It tracks all outputs (`-o`) the same way as `dvc add` does. Unlike

Reviewer: ```suggestion
`dvc run` writes a pipeline stage named `train` (specified using the `-n`
option) in [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file).
It tracks all outputs (`-o`) the same way as `dvc add` does. Unlike
```

Author: ![run](https://user-images.githubusercontent.com/46163196/86557482-e4439500-bf73-11ea-931e-02c2bac8dfd6.PNG)

Do we need to change it from here also?

Reviewer: Yes but it's out of scope. The whole ref. needs to be rewritten soon. I'm committing my suggestion above to save time. Please just fix the formatting. Resolving here.

Reviewer: Actually the command was already rewritten so yes, just this intro to that command should be rephrased... But still, out of scope for this PR. I'll send a separate one.

Reviewer: #1537

`dvc add`, `dvc run` also tracks dependencies (`-d`) and the command
(`python train.py`) that was run to produce the result.

> > At this point you could run `git add .` and `git commit` to save the updated
> > stage and its changed outputs to the repository.

Reviewer: ```suggestion
> At this point you could run `git add .` and `git commit` to save the `train`
> stage and its outputs to the repository.
```

> `dvc repro` will run `train` stage if any of its dependencies (`-d`) changed.

Reviewer: ```suggestion
`dvc repro` will run the `train` stage if any of its dependencies (`-d`) changed.
```

Reviewer: Please watch basic grammar.

> ```

```dvc
$ dvc run -n train \
          -d train.py -d data \
          -M metrics.csv \

>           -o model.h5 -o bottleneck_features_train.npy -o bottleneck_features_validation.npy \

Reviewer: Noticed this line is too long. I'll fix it:
```suggestion
          -o model.h5 \
          -o bottleneck_features_train.npy \
          -o bottleneck_features_validation.npy \
```

> ```

```dvc

> $ dvc run -n train \
>           -d train.py -d data \
>           -M metrics.csv \

Reviewer: ```suggestion
$ dvc run -n train -d train.py -d data \
```

>           -o model.h5 \
>           -o bottleneck_features_train.npy \
>           -o bottleneck_features_validation.npy \

Reviewer: ```suggestion
          -o model.h5 -o bottleneck_features_train.npy \
          -o bottleneck_features_validation.npy -M metrics.csv \
```
